### ISSUE TITLE
Mutt 1.7.2 segfaults instantly

### ISSUE THREAD

**Issue by Neo-Oli** [termux/termux-packages#587]
Backtrace
```
#0  0x0000007fb7fe1334 in __dl_$d () from /system/bin/linker64
#1  0x00000055710d33c4 in ?? ()
#2  0x0000007fffffcbf8 in ?? ()
#3  0xffff01f7ffffffff in ?? ()
Backtrace stopped: previous frame identical to this frame (corrupt stack?)

```

```
$ uname -a
Linux localhost 3.10.84-perf-g5c3f7a0 #1 SMP PREEMPT Mon Aug 1 14:14:49 2016 aarch64 Android

```

**Comment by Neo-Oli** [comment]
Also tested it on arm, also segfaults

**Comment by sp4rkie** [comment]
do you apply weird muttrc contents? Until now mutt didn't crash here. I use it very often on:
`Linux localhost 3.18.19+ #1 SMP PREEMPT Wed Nov 9 10:33:30 IST 2016 armv7l Android`

**Comment by Neo-Oli** [comment]
@sp4rkie No. I even tested it with a clean install of Termux. No config files at all. It's only a problem on the very latest version. 1.7.1 is working fine for me.

**Comment by sp4rkie** [comment]
you're right. 1.7.2 instantly crashes. After upgrading 2 packages:
```
Unpacking libcairo (1.14.8) over (1.14.6-3) ...
Unpacking mutt (1.7.2) over (1.7.1) ...
```

**Comment by Neo-Oli** [comment]
For now, you can downgrade to 1.7.1 again with 
`wget http://termux.net/dists/stable/main/binary-aarch64/mutt_1.7.1_aarch64.deb&&dpkg -i mutt_1.7.1_aarch64.deb`

**Comment by sp4rkie** [comment]
thanks!
`wget http://termux.net/dists/stable/main/binary-arm/mutt_1.7.1_arm.deb&&dpkg -i mutt_1.7.1_arm.deb`
in my case

**Comment by fornwall** [comment]
Thanks! This should be fixed by the 1.7.2-1 updated of the package now available in the apt repo.

I built the buggy package with an experimental ndk toolchain by mistake, but this update uses the normal one.

**Comment by sp4rkie** [comment]
Thanks a lot! Mutt on termux now running well again. It's a pleasure to run the basic unix/linux tools on a smartphone:-)
